components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -170,22 +170,13 @@ func display(v *policies_common.Version) string {
 }
 
 func runIAMUpgradeToV2Cmd(cmd *cobra.Command, args []string) error {
-	label := map[bool]string{
-		true:  "v2.1",
-		false: "v2",
-	}
 	upgradeReq := &policies_req.UpgradeToV2Req{
-		Flag:           policies_common.Flag_VERSION_2_0,
+		Flag:           policies_common.Flag_VERSION_2_1,
 		SkipV1Policies: iamCmdFlags.skipLegacyUpgrade,
 	}
 
-	isBetaVersion := iamCmdFlags.betaVersion
-	if isBetaVersion {
-		upgradeReq.Flag = policies_common.Flag_VERSION_2_1
-		writer.Title("Enabling IAM v2.1")
-	} else {
-		writer.Title("Upgrading to IAM v2")
-	}
+	writer.Title("Enabling IAM v2.1")
```

Comment:
Hmm... I think this needs to reflect the spirit of L187 not L185, so should change to "Upgrading"... not quite sure.

Reply:
Unintentional. Good catch.